src/components/organisms/OrganismHomeCard.vue

```diff
@@ -0,0 +1,155 @@
+                        <AtomIcon
+                            :icon="'car-back'"
+                            :size="'is-medium'"
+                        ></AtomIcon>
+                        <span> Car Owner </span>
+                    </div>
+                </template>
+                <template>
+                    <div class="card-main-body">
+                        <h2>Search a spot for the car</h2>
+                        <SearchInput class="mb-6"></SearchInput>
+                        <AtomButton @click.native="flyToSrp">
+                            Search now
+                        </AtomButton>
+                    </div>
+                </template>
+            </b-tab-item>
+            <b-tab-item>
+                <template #header>
+                    <div class="header-tab-btn">
+                        <AtomIcon
+                            :icon="'home-account'"
+                            :size="'is-medium'"
+                        ></AtomIcon>
+                        <span> Spot Owner </span>
+                    </div>
+                </template>
+                <template>
+                    <div class="card-main-body">
+                        <h2>Register unused parking spot</h2>
+                        <p class="mb-5">
+                            Start earning money by listing unused parking spot
+                            in our platform.
+                        </p>
+                        <AtomButton class="mb-5" @click.native="contactUs">
+                            Register now
+                        </AtomButton>
+                        <div>
+                            <span @click="contactUs">
+                                Learn more about being a spot owner.
+                            </span>
+                        </div>
+                    </div>
+                </template>
+            </b-tab-item>
+        </b-tabs>
+    </div>
+</template>
+
+<script>
+import AtomIcon from '../atoms/AtomIcon.vue';
+import SearchInput from '../extras/SearchInput.vue';
+import AtomButton from '../atoms/AtomButton.vue';
+import { mapGetters } from 'vuex';
+import { getCoordinate } from '../../includes/LatLng';
+export default {
+    name: 'HomeCard',
+    components: {
+        AtomIcon,
+        SearchInput,
+        AtomButton,
+    },
+    computed: {
+        ...mapGetters({
+            LocDetails: 'map/getLocDetails',
+        }),
+    },
+    methods: {
+        flyToSrp() {
+            const coordinate = getCoordinate(this.LocDetails.lnglat.toString())
+                .reverse()
+                .toString();
+            this.$router.push({
+                name: 'srp',
+                query: {
+                    latlng: coordinate,
+                },
+                params: {
+                    location: this.LocDetails.locDetails.locName,
+                },
+            });
+        },
+        onChange(tabNumber) {
+            if (tabNumber == 30) {
```

Comment:
why tabNumber==30?

Reply:
tabNumber is returned by the buefy component

Reply:
It is now changed